account_invoice_inter_company/models/account_move.py

```diff
@@ -68,9 +69,12 @@ def action_post(self):
                 src_invoice = src_invoice.with_user(intercompany_user).sudo()
             else:
                 src_invoice = src_invoice.sudo()
-            src_invoice.with_company(dest_company.id).with_context(
-                skip_check_amount_difference=True
-            )._inter_company_create_invoice(dest_company)
+            if not src_invoice.auto_generated:
+                src_invoice.with_company(dest_company.id).with_context(
+                    skip_check_amount_difference=True
+                )._inter_company_create_invoice(dest_company)
+            if src_invoice.filtered(lambda i: i.is_sale_document()):
+                src_invoice._attach_original_pdf_report()
```

Comment:
`src_invoice` is already unique.

```suggestion
            if src_invoice.is_sale_document():
                src_invoice._attach_original_pdf_report()
```